doc/qbk/3.0.parsing.qbk

```diff
@@ -68,7 +68,7 @@ URI partitioning:
     [`https://www.boost.org`]
     [URL]
 ][
-    [`mailto:mduerst@ifi.unizh.ch`]
+    [[teletype]`mailto:mduerst@ifi.unizh.ch`]
```

Comment:
No no, you can't use a real email address! Use `person@example.com`

Reply:
Weird. I didn't come up with that. That came from the RFC or something.